frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx

```diff
@@ -59,43 +75,50 @@ const NotificationEntry: React.FC<NotificationEntryProps> = ({
   targetPath,
   toggleNotificationDrawer,
   type,
-}) => (
-  <li
-    className={classNames(`pf-c-notification-drawer__list-item pf-m-hoverable pf-m-${type}`, {
-      'pf-m-read': isRead,
-    })}
-    tabIndex={0}
-    onClick={
-      targetPath
-        ? () => {
-            history.push(targetPath);
-            toggleNotificationDrawer();
-          }
-        : null
-    }
-  >
-    <div className="pf-c-notification-drawer__list-item-header">
-      <span className="pf-c-notification-drawer__list-item-header-icon">
-        <NotificationIcon type={type} />
-      </span>
-      <h4 className="pf-c-notification-drawer__list-item-header-title">
-        <span className="pf-screen-reader">{`${_.capitalize(type)} notification:`}</span>
-        {title}
-      </h4>
-      {actionText && actionPath && (
-        <NotificationAction
-          text={actionText}
-          path={actionPath}
-          onClick={toggleNotificationDrawer}
-        />
-      )}
-    </div>
-    <div className="pf-c-notification-drawer__list-item-description">{description}</div>
-    <div className="pf-c-notification-drawer__list-item-timestamp">
-      {timestamp && <Timestamp simple timestamp={timestamp} />}
-    </div>
-  </li>
-);
+}) => {
+  const { t } = useTranslation();
+  return (
+    <li
+      className={classNames(`pf-c-notification-drawer__list-item pf-m-hoverable pf-m-${type}`, {
+        'pf-m-read': isRead,
+      })}
+      tabIndex={0}
+      onClick={
+        targetPath
+          ? () => {
+              history.push(targetPath);
+              toggleNotificationDrawer();
+            }
+          : null
+      }
+    >
+      <div className="pf-c-notification-drawer__list-item-header">
+        <span className="pf-c-notification-drawer__list-item-header-icon">
+          <NotificationIcon type={type} />
+        </span>
+        <h4 className="pf-c-notification-drawer__list-item-header-title">
+          <span className="pf-screen-reader">
+            {t('notification-drawer~{{type}} notification:', {
+              type: notificationTypeString(type, t),
+            })}
```

Comment:
It would be better to make the entire string one message like `Warning notification:`. This is easier for the translators to localize.